src/actionlib/action_client.py

```diff
@@ -380,6 +385,16 @@ def _mark_as_lost(self):
         self.latest_goal_status.status = GoalStatus.LOST
         self.transition_to(CommState.DONE)
 
+    ## @brief Is the goal's age beyond the lost-message delta?
+    ##
+    ## @param status_timestamp Timestamp from the server status message.
+    ##
+    ## @return True if the goal is older than the delta, otherwise False
```

Comment:
This comment should be a Python docstring. However I would argue that it's completely superfluous, this method's function is simple enough that its name is explanation enough.